fdbrpc/HTTP.actor.cpp

```diff
@@ -530,32 +686,29 @@ ACTOR Future<Reference<HTTP::Response>> doRequest(Reference<IConnection> conn,
 ACTOR Future<Void> sendProxyConnectRequest(Reference<IConnection> conn,
                                            std::string remoteHost,
                                            std::string remoteService) {
-	state Headers headers;
-	headers["Host"] = remoteHost + ":" + remoteService;
-	headers["Accept"] = "application/xml";
-	headers["Proxy-Connection"] = "Keep-Alive";
 	state int requestTimeout = 60;
 	state int maxTries = FLOW_KNOBS->RESTCLIENT_CONNECT_TRIES;
 	state int thisTry = 1;
 	state double nextRetryDelay = 2.0;
 	state Reference<IRateControl> sendReceiveRate = makeReference<Unlimited>();
 	state int64_t bytes_sent = 0;
 
+	state Reference<OutgoingRequest> req;
+	req->verb = HTTP_VERB_CONNECT;
+	req->resource = remoteHost + ":" + remoteService;
+	req->data.content = nullptr;
+	req->data.contentLen = 0;
+	req->data.headers["Host"] = req->resource;
+	req->data.headers["Accept"] = "application/xml";
```

Comment:
do we always send data in `xml` format for this request?

Reply:
Thanks for adding the context.
I've seen silent failures due to application type deep in the stack and it maybe better for caller to decide it. Again, am not sure if such a flexibility makes sense in this usecase.